esque/controller/topic_controller.py

```diff
@@ -92,8 +66,14 @@ def create_topics(self, topics: List[Topic]):
             )
             future_list = self.cluster.confluent_client.create_topics([new_topic], operation_timeout=60)
             ensure_kafka_future_done(next(islice(future_list.values(), 1)))
+            for _ in range(80):
+                topic_data = self.cluster.confluent_client.list_topics(topic=topic.name).topics[topic.name]
+                if topic_data.error is None:
+                    break
+                time.sleep(0.125)
+            else:
```

Comment:
Wohoo. Finally a use for a for-else loop.

Reply:
They're really convenient for timeouts like this or for lookups :slightly_smiling_face:

Reply:
Yeah. The lookup use case is super handy.